Tools/ADT/MapsProtoTargeter.py

```
@@ -0,0 +1,459 @@
+                        if proto_value not in existing_ids:
+                            missing_ids.append(proto_value)
+
+    except FileNotFoundError:
+        print(f"Файл {file_path} не найден.")
+    except yaml.YAMLError as e:
+        print(f"Ошибка при обработке файла {file_path}: {e}")
+
+    if missing_ids:
+        output_file.write(f"{file_path} [{len(missing_ids)-1}]: {', '.join(missing_ids)}\n")
+    else:
+        output_file.write(f"{file_path}: Все прототипы найдены\n")
```

Ruff (0.12.2)

String contains ambiguous `В` (CYRILLIC CAPITAL LETTER VE). Did you mean `B` (LATIN CAPITAL LETTER B)?

(RUF001)